File: week1/Problem Set 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    while(true){
        height = get_int("Height: ");
        if(1 <= height && height <= 8) break;
    }
    int r = height;


    for(int i = 0; i < height; i ++){
        r --;

        for(int j = 0; j < r; j ++){
            printf(" ");
        }
        for(int j = 0; j < i + 1; j ++){
            printf("#");
        }
        printf("  ");
        for(int j = 0; j < i + 1; j ++){
            printf("#");
        }
        printf("\n");
    }

}

File: week6/Problem Set 6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if h <= 8 and h >= 1:
        break

for i in range(h):
    print(" " * (h - i - 1) + "#" * (i + 1) + "  " + "#" * (i + 1))